src/frontend/src/actions/profile.js

```diff
@@ -31,9 +31,7 @@ export const setProfile = (profile) => (dispatch, getState) => {
 
 export const sendLoginLink = (email) => (dispatch, getState) => {
   addActionLogItem(`Sending login link to ${email}`);
-  axios.post("/api/login/", {
-    email: email,
-  });
+  axios.post("/api/login/", { email });
```

Comment:
Just double-checking, things like this will cause CI to fail once #457 is merged, right?

Reply:
Good question. The short answer is no for the existing app code, but yes for the future frontend app code. I updated the `format` script, and such things should be autocorrected by that script. For example, this change was made automatically by the format script. I already ran the formatting and linting command with fixes in the bundler PR. 

So, how about I add a checkbox item in the PR template to suggest running the `./scripts/format`?

Reply:
That sounds good to me! It'll help people to remember to run `scripts/format` even if CI is going to catch any problems, so it may save some time.

Reply:
Agreed!

Reply:
Addressed here: https://github.com/azavea/echo-locator/pull/500/commits/262686ecae83a0af62538305cebe8a917f020884